src/App.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import { Toaster } from 'react-hot-toast';
+import { NavigationProvider } from './components/NavigationContext';
+import Navigation from './components/Navigation';
+import CenterContainer from './components/styledcomponents/CenterContainer';
+import Reservations from './components/Reservations';
+import Aeroplanes from './components/Aeroplanes';
+import './App.css';
+import Reserve from './components/Reserve';
+import AddAeroplane from './components/AddAeroplane';
+import DeleteAeroplane from './components/DeleteAeroplane';
+import Login from './components/Login';
+import Register from './components/Register';
+import RequireAuth from './components/RequireAuth';
+import AeroPlaneDetails from './components/AeroPlaneDetails';
+import LoginStyle from './components/styledcomponents/LoginStyle';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 9):


![image](https://github.com/rubydevi/jetlogix-frontend/assets/57421823/2b43cad2-7fb8-4543-bf71-7d9567eb6805)

That way, your code will meet the latest standards. 💪